server/tserver/src/main/java/org/apache/accumulo/tserver/InMemoryMap.java

```diff
@@ -754,9 +758,11 @@ public void delete(long waitTime) {
           aconf = createSampleConfig(aconf);
         }
 
+        // TODO can we drop the crateSampleConfig and just use tableConf?
```

Comment:
```suggestion
        // TODO can we drop the createSampleConfig and just use tableConf?
```

Reply:
I added this comment to look at this again and I don't think we can. Removed comment in 515e963.